addons/block_code/inspector_plugin/block_script_inspector.gd

```diff
@@ -1,5 +1,7 @@
 extends EditorInspectorPlugin
 
+const BlockCodePlugin = preload("res://addons/block_code/block_code_plugin.gd")
```

Comment:
The reason for this is actually pretty ugly. It's to get access to the `BlockCodePlugin.main_panel` static singleton. It would be better to pass in the handler so that's not needed. I started working on it but set it aside for the moment.